released-versions.json

```diff
@@ -0,0 +1,224 @@
+        },
+        {
+            "buildTime": "20160815131501+0000",
+            "version": "3.0"
+        },
+        {
+            "buildTime": "20160718063837+0000",
+            "version": "2.14.1"
+        },
+        {
+            "buildTime": "20160614071637+0000",
+            "version": "2.14"
+        },
+        {
+            "buildTime": "20160425041010+0000",
+            "version": "2.13"
+        },
+        {
+            "buildTime": "20160314083203+0000",
+            "version": "2.12"
+        },
+        {
+            "buildTime": "20160208075916+0000",
+            "version": "2.11"
+        },
+        {
+            "buildTime": "20151221211504+0000",
+            "version": "2.10"
+        },
+        {
+            "buildTime": "20151117070217+0000",
+            "version": "2.9"
+        },
+        {
+            "buildTime": "20151020034636+0000",
+            "version": "2.8"
+        },
+        {
+            "buildTime": "20150914072616+0000",
+            "version": "2.7"
+        },
+        {
+            "buildTime": "20150810131506+0000",
+            "version": "2.6"
+        },
+        {
+            "buildTime": "20150708073837+0000",
+            "version": "2.5"
+        },
+        {
+            "buildTime": "20150505080924+0000",
+            "version": "2.4"
+        },
+        {
+            "buildTime": "20150216050933+0000",
+            "version": "2.3"
+        },
+        {
+            "buildTime": "20141124094535+0000",
+            "version": "2.2.1"
+        },
+        {
+            "buildTime": "20141110133144+0000",
+            "version": "2.2"
+        },
+        {
+            "buildTime": "20140908104039+0000",
+            "version": "2.1"
+        },
+        {
+            "buildTime": "20140701074534+0000",
+            "version": "2.0"
+        },
+        {
+            "buildTime": "20140429092431+0000",
+            "version": "1.12"
+        },
+        {
+            "buildTime": "20140211113439+0000",
+            "version": "1.11"
+        },
+        {
+            "buildTime": "20131217092815+0000",
+            "version": "1.10"
+        },
+        {
+            "buildTime": "20131119082002+0000",
+            "version": "1.9"
+        },
+        {
+            "buildTime": "20130924073233+0000",
+            "version": "1.8"
+        },
+        {
+            "buildTime": "20130806111956+0000",
+            "version": "1.7"
+        },
+        {
+            "buildTime": "20130507091214+0000",
+            "version": "1.6"
+        },
+        {
+            "buildTime": "20130327140935+0000",
+            "version": "1.5"
+        },
+        {
+            "buildTime": "20130128034246+0000",
+            "version": "1.4"
+        },
+        {
+            "buildTime": "20130125035146+0000",
+            "version": "1.4-rc-3"
+        },
+        {
+            "buildTime": "20130122030632+0000",
+            "version": "1.4-rc-2"
+        },
```

Comment:
❌  should not be in this list I guess:
- `1.2-rc-1`
- `1.4-rc-2`
- `1.4-rc-3`

Reply:
Fixed!